resources/build/version/src/fixupHistory.ts

```diff
@@ -223,17 +223,19 @@ export const sendCommentToPullRequestAndRelatedIssues = async (
   // https://docs.github.com/en/rest/guides/best-practices-for-integrators#dealing-with-abuse-rate-limits
```

Comment:
This was not intended to come along in this PR but it doesn't hurt. Just cleans up the logging in the nightly build trigger to show the correct pull request log entry with the correct timing (otherwise all the log entries are shown at once, but there is still a 10 sec delay between comment submissions)